p2p/src/main/kotlin/maru/p2p/discovery/MaruDiscoveryService.kt

```diff
@@ -202,10 +207,11 @@ class MaruDiscoveryService(
   }
 
   private fun createLocalNodeRecord(): NodeRecord {
+    val sequenceNumber = runtimeConfigs.getDiscoverySequenceNumber() + 1uL
```

Comment:
The localNodeRecordUpdated is only called when the record is changed, not when a new LocalNodeRecordStore is created. That means we have to store that new sequence number here as well.